source/MBF.SAMD51.GPIO.pas

```diff
@@ -155,6 +241,20 @@ procedure TGPIO.Initialize;
 begin
 end;
 
+{$ifdef freertos_fat}
+procedure freertos_pinMode(ulPin:DWORD;ulMode:DWORD); external name 'pinMode';
```

Comment:
please do not define externals here for code that should go to adc unit, this will create pain lateron because it breaks the whole structure of mbf, adc stuff goes in the adc.
When you need externals from a wio specific lib then make it clear that it comes from wio, not freertos, freertos itself does not provide those functions....

Reply:
Agreed. Please inspire me in finding a right naming scheme for these external functions.